pdc/apps/common/viewsets.py

```diff
@@ -205,7 +205,7 @@ def _check_ordering_keys(self, request):
         serializer_fields = self._get_fields_from_serializer_class()
         valid_fields = list(set(model_fields).intersection(set(serializer_fields)))
         valid_fields += self.queryset.query.aggregates.keys()
-        tmp_list = [param.strip().lstrip('-') for param in ordering_keys.split(',')]
+        tmp_list = [param.strip().lstrip('-').split("__")[0] for param in ordering_keys.split(',')]
```

Comment:
As I mentioned here, if there is no this changes, if I call <server_url>/?orderging=to_component__name.
 It will raise error like:

Request Method: | GET
-- | --
http://10.66.128.226:8080/rest_api/v1/release-component-relationships/?ordering=-to_component__name
1.8.18
FieldError
Unknown query key: [u'to_component__name'] not in fields: ['relation_type', 'id', 'from_component', 'to_component']

I think this function is just check if the fields in model are correct. And if it includes nested files with "__", will check it in the funciton  is_valid_field of RelatedNestedOrderingFilter.

So after this change:
if you use a url like:
<server_url>/?orderging=to_componentsadfsa
It will raise errors:

Request Method: | GET
-- | --
http://10.66.128.226:8080/rest_api/v1/release-component-relationships/?ordering=-to_componentasdf

And if you use:
<server_url>/?orderging=to_component__namesadfsa.
will raise error like:

Request Method: | GET
-- | --
http://10.66.128.226:8080/rest_api/v1/release-component-relationships/?ordering=-to_component__nameasdfads
1.8.18